src/harness/testcases/WINNF_FT_S_PCR_testcase.py

```diff
@@ -200,15 +209,15 @@ def test_WINNF_FT_S_PCR_1(self, config_filename):
     self.assertIsNotNone(ppa_id)
 
     # Notify the SAS UUT about the SAS Test Harness.
-    certificate_hash = getCertificateFingerprint(config['serverCert'])
+    certificate_hash = getCertificateFingerprint(config['sasTestHarnessCert'])
     self._sas_admin.InjectPeerSas({'certificateHash': certificate_hash,
                                    'url': self._sas._base_url})
```

Comment:
Why are you using the SAS's own URL? This makes no sense.

Reply:
We have   updated accordingly to pass SAS Test Harness URL here